src/main/java/ru/skypro/homework/controller/AdsController.java

```diff
@@ -32,8 +32,8 @@ public ResponseEntity<Ads> getAllAds(Authentication authentication){
                 .body(allAds);
     }
 
-    @PostMapping()
-    public ResponseEntity<Ad> addAd(Authentication authentication, @RequestBody @Valid CreateOrUpdateAd properties, @RequestBody MultipartFile image){
+    @PostMapping(consumes = MediaType.MULTIPART_FORM_DATA_VALUE)
+    public ResponseEntity<Ad> addAd(Authentication authentication, @RequestBody @Valid CreateOrUpdateAd properties, @RequestPart MultipartFile image){
```

Comment:
```suggestion
    public ResponseEntity<Ad> addAd(Authentication authentication, @RequestPart @Valid CreateOrUpdateAd properties, @RequestPart MultipartFile image){
```
для properties нужна тоже аннотация RequestPart